modules/public/public.go

```diff
@@ -99,6 +106,20 @@ func (opts *Options) handle(ctx *macaron.Context, log *log.Logger, opt *Options)
 
 	f, err := opt.FileSystem.Open(file)
 	if err != nil {
+		// 404 requests to any known child directories under `public`
+		if path.Base(opts.Directory) == "public" {
+			parts := strings.Split(file, "/")
+			if len(parts) < 2 {
+				return false
+			}
+			for _, dir := range resourceDirs {
+				if dir == parts[1] {
+					ctx.Resp.WriteHeader(404)
+					ctx.Resp.Write([]byte(""))
```

Comment:
This line may be unneccesary, I just wanted to ensure no further processing is done.